tests/test-slirp4netns-configure.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xeuo pipefail
+
+. $(dirname $0)/common.sh
+
+unshare -r -n sleep 1000 &
+child=$!
+
+wait_for_network_namespace $child
+
+slirp4netns -c $child tun11 &
+slirp_pid=$!
+
+wait_for_network_device $child tun11
+
+function cleanup {
+    kill -9 $child $slirp_pid
+}
+trap cleanup EXIT
+
+nsenter --preserve-credentials -U -n --target=$child ifconfig -a | grep tun11
```

Comment:
Maybe we can use `ip -j` and `jq`?
`ip -j` might not be available in older distros, but not a severe problem if we containerize the test env.